### DISCUSSION TITLE
Installed perfectly and worked at the time... But now HA doesnt update the dashboard

### DISCUSSION THREAD

**Discussion by Judge-Tread** [greghesp/ha-bambulab#155] [Q&A]
Installed and all worked perfectly at the time - HA dashboard reflected what was being shown on the Bambu Slicer while printing. 

However its now not updating HA, and the only thing that has happened is I have powered off the printer (using the switch on the back) as it's standby consumption is very high (due to keeping nozzle/bed at a set temperature for quicker startup when printing).

Does powering off change the Access code? or could it merely have just been allocated a new IP from the router?

**Comment by simoniddqd** [comment]
Have the same problem, although I havent turned off the printer

**Comment by greghesp** [comment]
Could you let us know if this is stil an issue please?

**Comment by Judge-Tread** [comment]
Still no joy getting this to update Sent from my iPhoneOn 27 Sep 2023, at 00:58, Greg Hesp ***@***.***> wrote:﻿
Could you let us know if this is stil an issue please?

**Reply by AdrianGarside** [reply]
Have you upgraded the integration to the latest version? There's been many bug fixes since June that improve mqtt connection reliability.

But if you're running a P1P/S, be aware of this firmware issue:
https://github.com/greghesp/ha-bambulab/issues/174

**Comment by Judge-Tread** [comment]
Ooh I am not - I shall retract my grumbles and go do some updates!Sent from my iPhoneOn 27 Sep 2023, at 17:45, AdrianGarside ***@***.***> wrote:﻿
Have you upgraded the integration to the latest version? There's been many bug fixes since June that improve mqtt connection reliability.
But if you're running a P1P/S, be aware of this firmware issue:
#174

**Comment by greghesp** [comment]
Closing. If it's still a problem after updating, please can you open an issue
